### PR TITLE
Ignore Playwright setup errors for newer runner images

### PR DESCRIPTION
**Pull request by danielmx-dev**
Fixes #

#### Changes proposed in this Pull Request

 WooPayments PW E2E tests are  failing due to the Ubuntu 24.04 image change. Particularly this step
```
- name: Kill mono-xsp4.service
      shell: bash
      run : |
        echo "::group::Kill webserver running on port 8084"
        sudo fuser -k -n tcp 8084
        echo "::endgroup::"
```

It looks like Mono is not included anymore, if we compare the list of included software between images: [24](https://github.com/actions/runner-images/blob/ubuntu24/20241006.1/images/ubuntu/Ubuntu2404-Readme.md) vs [22](https://github.com/actions/runner-images/blob/ubuntu22/20241006.1/images/ubuntu/Ubuntu2204-Readme.md)

#### Testing instructions

* Run PW E2E tests using this branch, they should pass. Verify that the runner image is set to 
```
Image: ubuntu-24.04
Version: 20241006.1.0
```

-------------------

- [ ] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

File: .github/actions/e2e/env-setup/action.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ runs:
       shell: bash
       run : |
         echo "::group::Kill webserver running on port 8084"
-        sudo fuser -k -n tcp 8084
+        sudo fuser -k -n tcp 8084 || true
```
Comment (danielmx-dev) [inline .github/actions/e2e/env-setup/action.yml:69]: According to [the manual](https://man7.org/linux/man-pages/man1/fuser.1.html)
> fuser returns a non-zero return code if none of the specified files is accessed

So we can just ignore if the port is not busy.

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `9557` or branch name `fix/playwright-e2e-environment-setup` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=fix/playwright-e2e-environment-setup) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: 17baaae687878f97908b0d8611ab11b974d12d19
- Build time: 2024-10-15 15:28:07 UTC

_Note: the build is updated when a new commit is pushed to this PR._

**Review by achyuthajoy — APPROVED** [review]
LGTM 🚢

**Comment by danielmx-dev** [comment]
Here's a [successful run](https://github.com/Automattic/woocommerce-payments/actions/runs/11349258736/job/31564936384) in Ubuntu 24

image

**Pull request merged**